docs/contribute/development-environment.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Saros Development Environment
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+The following page describes how a development environment can be set up in order to develop all Saros products (Eclipse-Plugin, IDEA-Plugin and Server).
+If you want to execute the STF tests it is recommended to use eclipse. Otherwise it is also possible to develop with IntelliJ IDEA and execute the stf tests in a docker container. 
+
+## Common
+* You have to [clone](https://help.github.com/articles/cloning-a-repository/) the Saros repository with [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+* You need a **Java 8 JDK**.
+* You need an **Eclipse 4.6** installation which is used for dependency resolution. You can either install a [minimal Eclipse](http://www.eclipse.org/downloads/packages/release/neon/3/eclipse-ide-java-developers) or [Eclipse for Eclipse Committers](http://www.eclipse.org/downloads/packages/release/neon/3/eclipse-ide-eclipse-committers) if you also want to develop with eclipse.
```

Comment:
Normally you need the Eclipse Version with PDE support. I do not know if your Gradle changes made this obsolete.

Reply:
This is not obsoleted by gradle. The "Eclipse IDE for Eclipse Committers" is the eclipse version that contains the PDE plugin.